react-components/src/components/LoadingIndicator/indicator.js

```diff
@@ -20,13 +20,7 @@ const LoadingIndicator = props => {
 
     return (
         <div className={className}>
-            <img
-                className={classes.indicator}
-                src="/conf/venia/settings/wcm/templates/product-page/structure/_jcr_content/root/header/logo.svg"
-                width="64"
-                height="64"
-                alt="Loading indicator"
-            />
+            <span className={`${classes.indicator} loader-img`}></span>
```

Comment:
Shouldn't this simply be `<span className=${classes.indicator} loader-img></span>`? (without the extra `${...}` variable and backticks). BTW, shouldn't we have quotes for all these attributes like `<span className="${classes.indicator} loader-img"></span>` or are these automatically added by React?

Reply:
@cjelger `className={`${classes.indicator} loader-img`}` is correct.

Reply:
@cjelger the outer `{` delimit a javascript block code inside the JSX